src/page/registration/registration.tsx

```diff
@@ -14,79 +14,96 @@ import {styles} from './registrationCss';
 import {FontAwesomeIcon} from '@fortawesome/react-native-fontawesome';
 import {faArrowLeft} from '@fortawesome/free-solid-svg-icons';
 import Logo from '../../resource/images/logo.svg';
+import {useNavigation} from '@react-navigation/native';
+import LinearGradient from 'react-native-linear-gradient';
 
 const Registration: React.FC = () => {
+  const navigation = useNavigation();
   return (
     <KeyboardAvoidingView
       behavior={Platform.OS === 'ios' ? 'padding' : 'height'}
       enabled
       keyboardVerticalOffset={100}>
-      <ScrollView>
-        <TouchableOpacity
-          style={styles.container}
-          onPress={Keyboard.dismiss}
-          activeOpacity={1}>
-          <View style={styles.arrowWrapper}>
-            <FontAwesomeIcon
-              size={24}
-              color={'rgb(74, 74, 74)'}
-              icon={faArrowLeft}
-            />
-          </View>
-          <View style={styles.titleWrapper}>
-            <Text style={styles.title}>Ecommerce</Text>
-            <Text style={styles.title}>Store</Text>
-          </View>
-          <View style={styles.inputContainer}>
-            <View style={[styles.inputWrapper]}>
-              <TextInput
-                style={styles.textInput}
-                placeholder="Full Name"
-                placeholderTextColor="rgb(116, 116, 116)"
+      <LinearGradient
+        style={[{width: '100%', height: '100%'}]}
+        start={{x: 0, y: 0}}
+        end={{x: 0, y: 1}}
+        colors={[
+          'rgb(203, 220, 232)',
+          'rgb(206, 194, 213)',
+          'rgb(234, 214, 188)',
+        ]}>
+        <ScrollView>
+          <TouchableOpacity
+            style={styles.container}
+            onPress={Keyboard.dismiss}
+            activeOpacity={1}>
```

Comment:
It is not a good idea to set activeOpacity to 1 because of losing correct UX. 
use 0.8 i.e.